src/packages/react-force-graph-2d/package.json

```diff
@@ -9,6 +9,7 @@
   "module": "dist/react-force-graph-2d.mjs",
   "types": "dist/react-force-graph-2d.d.ts",
   "exports": {
+    "types": "./dist/react-force-graph-2d.d.ts",
```

Comment:
That's good, but could you please make the `exports` the last export, so you have `umd` and `default` first?

Reply:
The TS documentation explitly says that types need to come first:
> The "types" condition should always come first in "exports".

Reply:
Ok I see. Thanks for pointing that out. Seems a little heavy handed, but fine.